qujax/circuit_tools.py

```diff
@@ -1,4 +1,5 @@
-from typing import Sequence, Union, Callable, List, Tuple
+from __future__ import annotations
```

Comment:
Is this working with all python version you want to support?

Reply:
yep!